README.md

```diff
@@ -1,65 +1,117 @@
-
-[See our FAQ if you hit any problems getting setup](https://github.com/hotosm/tasking-manager/wiki/Dev-Environment-FAQ)
-
-**Looking to get involved in development?** Check out the [Working Group meeting details](https://github.com/hotosm/tasking-manager/wiki/TM-Working-Group-Meeting-Details), review the [Roadmap](https://github.com/hotosm/tasking-manager/projects/1), and review our [Contributor guidelines](https://github.com/hotosm/tasking-manager/blob/develop/CONTRIBUTING.md).
-
-## <a name="ClientDevelopment">Client Development</a>
-### Global Dependencies
-Following must be available locally:
-
-* NodeJS LTS v6+ [NodeJS LTS install here](https://nodejs.org/en/)
-* [Gulp](http://gulpjs.com/) is used to run and build the app, install globally as follows:
-    * ```npm install gulp -g```
-
-### App Dependencies
-You will now have to install all the app dependencies using [NPM](https://www.npmjs.com/)
+**Dependencies**
+The following dependencies must be available _globally_ on your system:
+* Download and install [NodeJS LTS v6+](https://nodejs.org/en/) and [NPM](https://www.npmjs.com/)
+* Install [Gulp](http://gulpjs.com/).
+  * `npm install gulp -g`
+* Install [Karma](https://karma-runner.github.io/1.0/index.html)
+  * `npm install -g karma karma-jasmine karma-chrome-launcher`
 
+**Build**
+Once you have the above dependencies, install and run the client dependencies using `npm` and `gulp`
 ```
 cd client
 npm install
+gulp run
 ```
 
-### Running Locally
-If you plan to do client development you can run the app using gulp, without having to worry too much about the server. If you want to point the client at a non-local API (e.g. a staging environment), you'll have to change the environment config for development in `client/taskingmanager.config.json`.
+**Tests**
+The client has a suite of [Jasmine](https://jasmine.github.io/) Unit Tests. These can be run using [Karma](https://karma-runner.github.io/1.0/index.html) as follows
 
 ```
-cd client   [if not already in client]
-gulp run
+ cd client
+ karma start ../tests/client/karma.conf.js
 ```
-
-### Running Unit Tests
-The client has a suite of [Jasmine](https://jasmine.github.io/) Unit Tests, that you can run using [Karma](https://karma-runner.github.io/1.0/index.html) as follows
-
+## Server
+The backend server is made up of a postgres database and an associated API that calls various end points to create tasks, manage task state, and produce analytics.
+
+**Dependencies**
+* [Python 3.6+](https://www.python.org/downloads/)
+  * Note: The project does not work with Python 2.x. You **will** need Python 3.x
+* [postgreSQL](https://www.postgresql.org/download/) with [postGIS](https://postgis.net/install/)
+* [pip](https://pip.pypa.io/en/stable/installing/)
+
+**Get secrets from OpenStreetMap**
+Since the tasking manager works with your OpenStreetMap credentials, register your local tasking manager application on OpenStreetMap to test advanced functionality.
```

Comment:
I would revise to `Since the tasking manager works with OpenStreetMap credentials, register your local tasking manager application on OpenStreetMap to allow logins.`